<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QMikAPIExample</class>
 <widget class="QMainWindow" name="QMikAPIExample">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1016</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QMikAPI example</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Usuario</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leUser">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Contraseña</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lePass">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Password</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>IP</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="leIP">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Puerto</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sbPort">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>65535</number>
          </property>
          <property name="value">
           <number>8728</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pbConnect">
          <property name="text">
           <string>Conectar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="autoScrollCheckBox">
          <property name="text">
           <string>mover barras auto.</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>50</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="lwResponses"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Sentence</string>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Etiqueta</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Los atributos, atributos API y los Queries sepárapos por comas cuando quieras enviar más de uno.</string>
         </property>
        </widget>
       </item>
       <item row="6" column="0" colspan="2">
        <widget class="QPushButton" name="pbEnviar">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Enviar</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="leCommand">
         <property name="text">
          <string>/tool/torch</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Queries</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>At. API</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Atributos</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Comando</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Si se envían queries, no se olvide en añadir el cálculo. Por ejemplo: #|</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QLineEdit" name="tagLineEdit">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="3">
        <widget class="QLineEdit" name="leAttrib">
         <property name="text">
          <string>interface=bridgeHotSpot,src-address=10.4.0.0/16</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="3">
        <widget class="QLineEdit" name="leAPI"/>
       </item>
       <item row="5" column="1" colspan="3">
        <widget class="QLineEdit" name="leQueries">
         <property name="text">
          <string>type=ether,#|</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leUser</tabstop>
  <tabstop>lePass</tabstop>
  <tabstop>leIP</tabstop>
  <tabstop>sbPort</tabstop>
  <tabstop>pbConnect</tabstop>
  <tabstop>autoScrollCheckBox</tabstop>
  <tabstop>leCommand</tabstop>
  <tabstop>tagLineEdit</tabstop>
  <tabstop>leAttrib</tabstop>
  <tabstop>leAPI</tabstop>
  <tabstop>leQueries</tabstop>
  <tabstop>pbEnviar</tabstop>
  <tabstop>lwResponses</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
